src/style/app-fixed.module.css

```diff
@@ -801,13 +809,58 @@
 .modalHeader {
   border: none;
   padding-bottom: 0;
-  background-color: var(--modalHeader-bg) !important;
-  padding: 1rem 1rem;
+  background-color: #f5f5f5 !important;
```

Comment:
Make this a variable. There may be suitable ones at the top of the file with the same color value.

Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`